packages/eslint-plugin/tests/rules/strict-enums/strict-enums-assignment.test.ts

```diff
@@ -0,0 +1,298 @@
+import type {
+  InvalidTestCase,
+  ValidTestCase,
+} from '@typescript-eslint/utils/src/ts-eslint';
+
+import type { MessageIds } from '../../../src/rules/strict-enums';
+import rule from '../../../src/rules/strict-enums';
+import {
+  fruit2EnumDefinition,
+  fruitEnumDefinition,
+  strictEnumsRuleTester,
+} from './strict-enums';
+
+const valid: ValidTestCase<unknown[]>[] = [];
+const invalid: InvalidTestCase<MessageIds, unknown[]>[] = [];
+
+valid.push({
```

Comment:
I don't think I've ever seen lint rule test files `.push` to `invalid` and/or `valid` before!